_rules/SC2-5-3-name-from-content-included-in-accessible-name.md

```diff
@@ -0,0 +1,176 @@
+---
+name: Name from content included in accessible name
+description: |
+  Interactive elements labelled through their content must have their visible label as part of their accessible name.
+
+success_criterion:
+- 2.5.3 # Label in Name
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any HTML or SVG element that:
+* has a [semantic role](#semantic-role) that is a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [supports name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent), and 
+* has [visible](#visible) [text](https://www.w3.org/TR/WCAG21/#dfn-text), and
+* has an [accessible name](#accessible-name), even if the name is the empty string ("").
```

Comment:
The reason why I started out doing this editing was because we on our side had a validation issue due to the "text" definition. So this problem is from the original rule - the off-screening was added due to additional findings in the validation. I am just trying to solve both issues in one PR...

Reply:
Wilco has agreed to keeping it in one rule, so I am marking this comment as resolved.

Reply:
"the empty string" should probably be just "empty" to align with https://github.com/act-rules/act-rules.github.io/pull/430.

Reply:
> "the empty string" should probably be just "empty" to align with #430.

Change done. I consider this change editorial, so therefore it will not impact the final review.